[2023-07-26T11:29:06.169-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LH_ED_GABRIELLEMUNIZ.extract_write_local manual__2023-07-26T14:23:45.992019+00:00 [queued]>
[2023-07-26T11:29:06.177-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LH_ED_GABRIELLEMUNIZ.extract_write_local manual__2023-07-26T14:23:45.992019+00:00 [queued]>
[2023-07-26T11:29:06.178-0300] {taskinstance.py:1308} INFO - Starting attempt 2 of 5
[2023-07-26T11:29:06.198-0300] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_write_local> on 2023-07-26 14:23:45.992019+00:00
[2023-07-26T11:29:06.203-0300] {standard_task_runner.py:57} INFO - Started process 18720 to run task
[2023-07-26T11:29:06.206-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'LH_ED_GABRIELLEMUNIZ', 'extract_write_local', 'manual__2023-07-26T14:23:45.992019+00:00', '--job-id', '411', '--raw', '--subdir', 'DAGS_FOLDER/LH_ED_GABRIELLEMUNIZ.py', '--cfg-path', '/tmp/tmpbp4ue_3c']
[2023-07-26T11:29:06.207-0300] {standard_task_runner.py:85} INFO - Job 411: Subtask extract_write_local
[2023-07-26T11:29:06.261-0300] {task_command.py:410} INFO - Running <TaskInstance: LH_ED_GABRIELLEMUNIZ.extract_write_local manual__2023-07-26T14:23:45.992019+00:00 [running]> on host ubuntu
[2023-07-26T11:29:06.363-0300] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabi' AIRFLOW_CTX_DAG_ID='LH_ED_GABRIELLEMUNIZ' AIRFLOW_CTX_TASK_ID='extract_write_local' AIRFLOW_CTX_EXECUTION_DATE='2023-07-26T14:23:45.992019+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-26T14:23:45.992019+00:00'
[2023-07-26T11:29:06.413-0300] {logging_mixin.py:150} WARNING - /home/gabi/Documentos/Gabi/Indicium/code-challenge-indicium/ind_env/lib/python3.10/site-packages/airflow/utils/context.py:313 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-07-26T11:29:06.466-0300] {python.py:183} INFO - Done. Returned value was: None
[2023-07-26T11:29:06.473-0300] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=LH_ED_GABRIELLEMUNIZ, task_id=extract_write_local, execution_date=20230726T142345, start_date=20230726T142906, end_date=20230726T142906
[2023-07-26T11:29:06.498-0300] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-26T11:29:06.518-0300] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
